### PR TITLE
ISPN-5598 Hit count computed incorrectly for getAll()

### PR DESCRIPTION
**Pull request by oscerd**
Hi,

This PR resolves ISPN-5598:
https://issues.jboss.org/browse/ISPN-5598

Bye,
Andrea

### CODE REVIEW

File: core/src/test/java/org/infinispan/jmx/CacheMgmtInterceptorMBeanTest.java [hunk 1]
```diff
@@ -151,6 +155,27 @@ public void testRemoves() throws Exception {
       assertRemoveHits(3);
       assertRemoveMisses(1);
    }
+   
+   public void testGetAll() throws Exception {
+      assertMisses(0);
+      assertHits(0);
+      assert 0 == advanced.getStats().getHits();
```
Comment (wburns) [inline core/src/test/java/org/infinispan/jmx/CacheMgmtInterceptorMBeanTest.java]: I know the other methods are using asserts, however we are trying to move over affected code to org.testng.AssertJUnit.* methods instead if you could.  Thanks.

**Comment by oscerd** [comment]
Hi, Yes. I'll change the testGetAll method to use testng.

Maybe, after changing the code of this test I can open a ticket to move the other tests to use testng. What do you think about?

**Comment by oscerd** [comment]
Ok, I've used org.testng.AssertJUnit.* and squashed the latest commit.

Thanks

**Comment by danberindei** [comment]
Integrated, thanks Andrea!

**Pull request closed without merging**